nion/swift/ExportDialog.py

```diff
@@ -254,6 +259,7 @@ def ok_clicked() -> bool:
             if path:
                 ui.set_persistent_string("export_directory", selected_directory)
                 display_shape = Geometry.IntSize(height=height_px, width=width_px)
+
```

Comment:
Unless there is a reason for it, remove this white space change. IMHO adding a blank line here doesn't add to readability. If anything, add a comment.